sdk/iotoperations/ci.mgmt.yml

```diff
@@ -0,0 +1,23 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+
+trigger: none
+pr:
+  branches:
+    include:
+    - main
+    - feature/*
+    - hotfix/*
+    - release/*
+  paths:
+    include:
+    - sdk/ProviderNameLowercase/ci.mgmt.yml
```

Comment:
Are these expected to say `ProviderNameLowercase` and not the actual service name?

Reply:
yes, you are right. I will quickly fix this. 
And this is a new repro [PR ](https://github.com/Azure/azure-sdk-for-net/pull/45916)with the issue you mentioned fixed.